docs/concepts/service-catalog/index.md

```diff
@@ -0,0 +1,233 @@
+---
+title: Service Catalog
+approvers:
+- chenopis
+---
+
+{% capture overview %}
+{% glossary_definition term_id="service-catalog" length="all" %}
```

Comment:
This honestly looks a little weird without the actual word "Service Catalog" in front of it. I understand the structure you're using to include things from the Glossary; maybe "Service Catalog is" should lead off the sentence, though. I don't know how you'd fix the odd capitalization.

Reply:
Yeah, we didn't want to make it redundant by using the word we were defining at the beginning of the sentence. Especially if we are using these snippets in tooltips.

How about "This extension API enables applications running in Kubernetes clusters to easily use external managed software offerings, such as a datastore service offered by a cloud provider."?

Reply:
Yeah, that's fine. It's just weird to start the sentence with "An."